public/app/features/correlations/CorrelationsPage.tsx

```diff
@@ -157,10 +158,17 @@ export default function CorrelationsPage() {
       navModel={navModel}
       subTitle={
         <>
-          Define how data living in different data sources relates to each other. Read more in the{' '}
-          <a href="https://grafana.com/docs/grafana/next/administration/correlations/" target="_blank" rel="noreferrer">
-            documentation <Icon name="external-link-alt" />
-          </a>
+          <Trans i18nKey="correlations.sub-title">
+            Define how data living in different data sources relates to each other. Read more in the
```

Comment:
An extra space will be needed before the link (not after). And translations should be re-generated, at the moment JSON contains:

```
    "sub-title": "Define how data living in different data sources relates to each other. Read more in the"
```

and the link to docs is not visible

<img width="803" alt="Screenshot 2023-09-28 at 13 06 32" src="https://github.com/grafana/grafana/assets/745532/f5443a1b-cf32-4031-86ae-72ff2d22bb1a">

Reply:
hey, I ran yarn i18n:extract but the JSON still container the same value. Do you know why it is not generating "documentation"

Reply:
Hmm... `i18n:extract` seems to be adding only new translations and does nothing when it already exists. Quite annoying during development of new translations :D I guess in such case, when it's the first time translation is added and only English translations are added you could probably revert all changes in grafana.json files (or remove the keys manually) and re-rerun `i18n:extract` - @grafana/grafana-frontend-platform is there a better way to do it?

Reply:
okay, just did it. lmk